clients/config.go

```diff
@@ -75,10 +77,22 @@ func (hc *HighwaterConfig) ToHostGetter(discovery disc.Discovery) disc.HostGette
 	return ToHostGetter("highwater", &hc.HostGetter, discovery)
 }
 
+// PortalConfig to use portal-api client
+type PortalConfig struct {
+	portal.Client
```

Comment:
I don't think we need the client interface here.
```suggestion
```

Reply:
well see